api/app/services/ingestions/ingestion_service.rb

```diff
@@ -0,0 +1,24 @@
+module Ingestions
+  class IngestionService < ActiveInteraction::Base
```

Comment:
I think this is actually an abstract interaction class and that should be made clear in the class name. Am I right in reading this as a service that doesn't do anything?